dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -130,7 +130,7 @@ if (process.env.LC_ALL) {
 }
 process.env.LC_NUMERIC = 'C';
 
-const uuid = require('uuid');
```

Comment:
Why was this change required?

Reply:
UUID no longer has a default export as of v7 or v8

Reply:
This is not making use of the default export, this is binding the `exports` object to the `uuid` name.